GameOfLife/GameOfLife/Iteration.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    public class Iteration
+    {
+        public List<int> cellsToDieX = new List<int>();
+        public List<int> cellsToDieY = new List<int>();
+
+        public List<int> cellsToBeBornX = new List<int>();
+        public List<int> cellsToBeBornY = new List<int>();
+
+        /// <summary>
+        /// Checks dead and alive cells according to the rules
+        /// </summary>
+        /// <param name="field"></param>
+        public void CheckCells(string[,] field)
+        {
+            int neigboursCountOfAlive = 0;
+            int neigboursCountOfDead = 0;
+            bool wrappedX = false;
+            bool wrappedY = false;
+
+            for (int i = 0; i < field.GetLength(0); i++)
+            {
+                for (int j = 0; j < field.GetLength(1); j++)
+                {
+                    if (field[i, j] == "X")
```

Comment:
This is too long method.
Make it having too calls to other methods. One new method will have logic for live cells and other for dead cells.
That is you move current logic into new methods.

Reply:
Split into two methods - CheckAlive() and CheckDead(), which are called using CheckCells() method.